Makefile

```diff
@@ -1,32 +1,55 @@
-.PHONY: help \
+.PHONY: clean requirements
 
 .DEFAULT_GOAL := help
 
 
-help: ## Display this help message
+help: ### Display this help message
 	@echo "Please use \`make <target>' where <target> is one of"
 	@perl -nle'print $& if m{^[a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-25s\033[0m %s\n", $$1, $$2}'
 
 clean: ## Remove generated byte code, coverage reports, and build artifacts
+	@echo "--> Clean Python files ..."
 	find . -name '__pycache__' -exec rm -rf {} +
 	find . -name '*.pyc' -exec rm -f {} +
 	find . -name '*.pyo' -exec rm -f {} +
 	find . -name '*~' -exec rm -f {} +
-	coverage erase
 	rm -fr build/
 	rm -fr dist/
 	rm -fr *.egg-info
 
-selfcheck: ## check that the Makefile is well-formed
+	@echo "--> Clean other files ..."
+	rm -fr var/
+	coverage erase
+	@echo "The project has been cleaned successfully."
+
+selfcheck: ## Check that the Makefile is well-formed
 	@echo "The Makefile is well-formed."
 
-requirements:
-	pip install -q -r requirements.txt --exists-action w
+test_requirements: ## Install requirements needed by test environment
+	pip install -q -r requirements/quality.txt --exists-action w
+	pip install -q -r requirements/test.txt --exists-action w
+
+requirements: test_requirements ## Installs all requirements needed by developmenent and test environments
+	pip install -q -r requirements/base.txt --exists-action w
 	pip install -e .
 	@echo "Finished installing requirements."
 
 quality:
-	pep8 html_xblock --max-line-length=120
-	pylint html_xblock
-
-test: quality ## Run tests in the current virtualenv
+	make selfcheck
```

Comment:
Can you add a comment for `quality` command? Otherwise it does not show in the commands list.